argparse/__init__.py

```diff
@@ -65,9 +65,26 @@ def load_conflicting_package(name, not_name, local_module):
 tools = []
 
 
+def get_arg2cwl_parser():
+    help_text = """
+    argparse2cwl forms CWL command line tool from Python tool
+    """
+    arg2cwl_parser = ap.ArgumentParser(prog=sys.argv[0], description=help_text, add_help=False)
+    arg2cwl_parser.add_argument('tools', nargs='*',
+                                help='Command for running your tool(s), without arguments')
+    arg2cwl_parser.add_argument('-d', '--directory',
+                                help='Directory for placing formed CWL tool')
+    arg2cwl_parser.add_argument('-b', '--basecommand',
+                                help='Command that appears in `basecommand` field in CWL tool')
```

Comment:
Again, is this a required option or does it override a normally correct default?